addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -6102,22 +6102,6 @@ legacyRegistry.GridColumns = SnippetOptionWidget.extend({
     },
 });
 
-legacyRegistry.vAlignment = SnippetOptionWidget.extend({
-    /**
-     * @override
-     */
-    async _computeWidgetState(methodName, params) {
-        const value = await this._super(...arguments);
-        if (methodName === 'selectClass' && !value) {
-            // If there is no `align-items-` class on the row, then the `align-
-            // items-stretch` class is selected, because the behaviors are
-            // equivalent in both situations.
-            return 'align-items-stretch';
-        }
-        return value;
-    },
```

Comment:
Why is this fully removed? Would be nice for the reason to be in the commit message 👍🏻